### PR TITLE
[FIX] run function was undifined

### PR DESCRIPTION
**Pull request by jeniaSakirko**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules/
+.idea/
```
Comment (tomas) [inline .gitignore]: This?

Comment (jeniaSakirko) [inline .gitignore]: Sorry, this line should be deleted I added it so It wouldn't add the WebStorm idea config to git

**Comment by Quanthir** [comment]
So will you merge this branch or fix the error?

**Comment by tomas** [comment]
@Quanthir I'm not really sure what the problem is or when it occurs, but I just pushed that change and published a new version.

**Comment by jeniaSakirko** [comment]
Hey guys sorry for the delayed answer. the problem was when calling the function "get_list" or "get_values"  an exception was thrown because the function "run" wan't defined.

**Pull request closed without merging**